aws/resource_aws_service_discovery_http_namespace_test.go

```diff
@@ -0,0 +1,92 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/servicediscovery"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSServiceDiscoveryHttpNamespace_basic(t *testing.T) {
+	resourceName := "aws_service_discovery_http_namespace.test"
+	rName := "terraformtesting"
```

Comment:
It'd be good to randomize this now, before additional parallel tests are added. 👍 

```suggestion
	rName := acctest.RandomWithPrefix("tf-acc-test")
```